osu.Game/Screens/Select/Carousel/DrawableCarouselBeatmap.cs

```diff
@@ -194,15 +192,15 @@ protected override void Deselected()
 
         protected override bool OnClick(ClickEvent e)
         {
-            if (Item.State.Value == CarouselItemState.Selected)
+            if (Item!.State.Value == CarouselItemState.Selected)
```

Comment:
Unsure about this also. In tests I can look away, but I don't think we want to set precedent that it's fine to use null-forgiveness liberally in non-test code.

`Item?.State.Value == ...` would feel more correct to me here. Same comment will probably also apply to 20 other places, so I'll spare everyone the tedium of repeating myself - applies everywhere.

Reply:
yeah i just didnt know how to handle this i didnt feel good about supressing everywhere